kernel/run.py

```diff
@@ -372,23 +382,19 @@ def pmem_paths(args):
                     paths.append(default)
                     pmem = "on"
                 else:
-                    if len(paths) > 0:
-                        paths.append(paths[node % num_host_numa_nodes])
-                    else:
-                        f = open("{}".format(pmem_test_path), "w+")
-                        paths += [pmem_test_path] * args.qemu_nodes
-                        return paths
+                    path = "{}-{}".format(pmem_test_path, node)
+                    open(path, "w+")
```

Comment:
what does this line do? return result is not stored anywhere?

Reply:
Thank you for the fantastic review. It was helpful.

I completely changed the run.py PMem related code. Not applicable anymore.